Sources_v3/APIClient/Endpoints/ChannelEndpoints.swift

```diff
@@ -107,4 +107,14 @@ extension Endpoint {
             body: ["remove_members": userIds]
         )
     }
+    
+    static func markRead(cid: ChannelId) -> Endpoint<EmptyResponse> {
+        .init(
+            path: "channels/\(cid.type)/\(cid.id)/read",
+            method: .post,
+            queryItems: nil,
+            requiresConnectionId: false,
+            body: nil
+        )
+    }
```

Comment:
This should be tested

Reply:
Fixed in d88845270f2dfe90ce3f887da7d256f728e1fb0e